### PR TITLE
Added usage for slf4j

### PR DESCRIPTION
**Pull request by benor1470**
Added slf4j as a fix for issue #63

### CODE REVIEW

**Comment by brettwooldridge** [comment]
@bhamiltoncx Does a dependency on slf4j cause Buck any grief?  Is java.util.logging better (no external dependency), allowing users to use jul-to-slf4j.jar if they so choose?

**Comment by benor1470** [comment]
SLF4J provides much simple and powerful APIs when compared to Java Util Logger and it is a standard

**Comment by brettwooldridge** [comment]
@benor1470 I understand it is widely used, I use it in many projects.  The issue is more of not breaking existing users who may not have a slf4j dependency.

**Comment by bhamiltoncx** [comment]
Buck doesn't really want slf4j as a dependency. We use java.util.logging
with a thin shim (the same as Airlift) and it works fine.

**Comment by benor1470** [comment]
Then I removed slf4j and used native java

File: pom.xml [hunk 1]
```diff
@@ -110,7 +110,7 @@
                        <Bundle-Name>NuProcess</Bundle-Name>
                        <Export-Package>com.zaxxer.nuprocess</Export-Package>
                        <Export-Package>com.zaxxer.nuprocess.codec</Export-Package>
-                       <Import-Package>com.sun.*,sun.misc.*</Import-Package>
+                       <Import-Package>com.sun.*,sun.misc.*,org.slf4j</Import-Package>
```
Comment (brettwooldridge) [inline pom.xml]: This change seems unnecessary.  I am ready to accept this pull request after this is removed.

**Comment by analytically** [comment]
There's always `jul-to-slf4j`: 

The jul-to-slf4j module includes a java.util.logging (jul) handler, namely SLF4JBridgeHandler, which routes all incoming jul records to the SLF4j API. 

http://www.slf4j.org/legacy.html

**Comment by bturner** [comment]
It's nice to see this getting tweaked for `WindowsProcess`, but shouldn't the same change be made to the `printStackTrace` calls in `BasePosixProcess` and `ProcessCompletions` (which is also in the Windows-specific code) and others? This seems like a pretty incomplete change.

**Comment by brettwooldridge** [comment]
Static loggers in a library is an [anti-pattern](http://wiki.apache.org/commons/Logging/StaticLog).

**Pull request closed without merging**